server.js

```diff
@@ -1481,6 +1481,73 @@ cache(function(data, match, sendBadge, request) {
           badgeData.colorscheme = 'blue';
         }
         sendBadge(format, badgeData);
+      } else if (info == 'wheel') {
+        var releases = data.releases[data.info.version];
+        var hasWheel = false;
+        for (var i in releases) {
+          if (releases[i].packagetype == 'wheel' ||
+              releases[i].packagetype == 'bdist_wheel') {
+            hasWheel = true;
+            break;
+          }
+        }
+        badgeData.text[0] = 'wheel';
+        badgeData.text[1] = hasWheel ? 'yes' : 'no';
+        badgeData.colorscheme = hasWheel ? 'brightgreen' : 'red';
+        sendBadge(format, badgeData);
+      } else if (info == 'py_versions') {
+        var versions = [];
+        var pattern = /^Programming Language \:\: Python \:\: (\d\.\d)$/;
+        for (var i in data.info.classifiers) {
+          var matched = pattern.exec(data.info.classifiers[i]);
+          if (matched && matched[1]) {
+            versions.push(matched[1]);
+          }
+        }
+        if (!versions.length) {
+          versions.push('not found');
+        }
+        badgeData.text[0] = 'python';
+        badgeData.text[1] = versions.sort().join(', ');
+        badgeData.colorscheme = 'blue';
+        sendBadge(format, badgeData);
+      } else if (info == 'implementation') {
+        var implementations = [];
+        var pattern = /^Programming Language \:\: Python \:\: Implementation \:\: (\S+)$/;
+        for (var i in data.info.classifiers) {
```

Comment:
Could you use a regular loop here, please?
```js
for (var i = 0; i < data.info.classifiers.length; i++) {
…
}
```

(Yes, I agree, JS is ugly compared to Python ☺.)